docs/using.md

```diff
@@ -60,6 +60,21 @@ Another common example might be the URL of the ISO to be used. Usually you speci
 _iso-url_ of the [start](./docs/minishift_start.md) command. Applying the rules from above, you can
 also specify this URL by setting the environment variable _MINISHIFT_ISO_URL_.
 
+Minishift also maintain a config file (_~/.minishift/config/config.json_) which can be used to set some
```

Comment:
I would prefer to document it instead auto-generating it then user try to edit manually instead of `set` sub-command but if we document it how to generate it using `set` sub-command then user have mindset to use it to setup a property and it will be less error-prone.